src/components/Form/FormFieldWerkingsgebied/FormFieldWerkingsgebied.tsx

```diff
@@ -163,6 +171,7 @@ const CardSelectedWerkingsgebied = ({
                     <span
                         className="absolute bottom-0 left-0 px-5 py-5 text-sm text-red-600 underline transition-colors duration-100 ease-in cursor-pointer hover:text-red-800"
                         onClick={() => {
+                            console.log('ONTKOPPEL!')
```

Comment:
@Aidenbuis Is this a test console.log? :P Otherwise change it to something more descriptive

Reply:
Woops, that one sneaked in 🥷🏻